backend/endpoints/messages.py

```diff
@@ -232,7 +233,7 @@ async def get_messages(org_id, room_id):
             status_code=status.HTTP_424_FAILED_DEPENDENCY,
             detail="Zc Core failed",
         )
-    return JSONResponse(
-        content=ResponseModel.success(data=response, message="Messages retrieved"),
-        status_code=status.HTTP_200_OK,
-    )
+    return paginate(response)
```

Comment:
Well done @man-of-mind it seems there isnt a limit for the paginated results specified in this PR. What's the FastAPI default?

Reply:
The default is 50, I can add custom limit if it is required.

Reply:
Oh since there's a default, it's fine 👍🏻